lib/phlex/rails/helpers.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 module Phlex::Rails::Helpers
+	def self.helpers
+		constants.map { |constant| const_get constant }
```

Comment:
We should probably exclude [`Phlex::Rails::Helpers::Tag`](https://github.com/phlex-ruby/phlex-rails/blob/main/lib/phlex/rails/helpers/tag.rb). I wonder if we shouldn’t just update `helpers.rb` to do the including, e.g.

```ruby
module Phlex::Rails::Helpers
	module ActionCableMetaTag
		extend Phlex::Rails::HelperMacros

		# @!method action_cable_meta_tag
		# 	Outputs an "action-cable-url" meta tag with the value of the URL specified in your configuration. Ensure this is above your JavaScript tag:
		# 	@see https://api.rubyonrails.org/classes/ActionCable/Helpers/ActionCableHelper.html#method-i-action_cable_meta_tag
		# 	@return [nil]
		define_output_helper :action_cable_meta_tag
	end

  include ActionCableMetaTag

  # ...
```

Reply:
Does it even make sense to keep these modules separate? Rails doesn't, so I'd be inclined to get rid of the individual modules and move all of the `define_output_helper :video_tag`, etc. declarations into the `Helpers` namespace.